ga4gh/cli.py

```diff
@@ -414,14 +415,18 @@ def __init__(self, args):
         self._referenceId = args.id
         self._start = args.start
         self._end = args.end
+        self._outputFormat = args.outputFormat
 
     def run(self):
-        iterator = self._client.listReferenceBases(
+        sequence = self._client.listReferenceBases(
             self._referenceId, self._start, self._end)
-        # TODO add support for FASTA output.
-        for segment in iterator:
-            print(segment, end="")
-        print()
+        if self._outputFormat == "text":
+            print(sequence)
+        else:
+            start = self._start if self._start else ""
+            end = self._end if self._end else ""
+            print(">{}:{}-{}".format(self._referenceId, start, end))
+            print(textwrap.fill(">{}".format(sequence)), sep="")
```

Comment:
Unfortunately, it turns out this is _very_ slow, so we have to do the splitting by hand. Something like
```
          width = 70
            chunks = len(sequence) // width
            for j in range(chunks):
                print(sequence[j * width : (j + 1) * width])
            if len(sequence) % width != 0:
                print(sequence[chunks * width:])
```
should work.

Of course, we really should add some tests for this. Can we add an issue stating that we need some tests for the FASTA sequence output?